src/client/common/application/commands.ts

```diff
@@ -125,10 +125,11 @@ export interface ICommandNameArgumentTypeMapping extends ICommandNameWithoutArgu
     [DSCommands.GotoPrevCellInFile]: [];
     [DSCommands.ScrollToCell]: [Uri, string];
     [DSCommands.ViewJupyterOutput]: [];
-    [DSCommands.ExportAsPythonScript]: [INotebookModel, PythonEnvironment | undefined];
-    [DSCommands.ExportToHTML]: [INotebookModel, string | undefined, PythonEnvironment | undefined];
-    [DSCommands.ExportToPDF]: [INotebookModel, string | undefined, PythonEnvironment | undefined];
-    [DSCommands.Export]: [Uri | INotebookModel, string | undefined, PythonEnvironment | undefined];
+    [DSCommands.ExportAsPythonScript]: [INotebookModel | undefined, PythonEnvironment | undefined];
```

Comment:
Correcting types. undefined is allowed for any of these when calling from the command palette.